src/Infrastructure/BotSharp.Abstraction/Conversations/Models/Conversation.cs

```diff
@@ -6,6 +6,7 @@ public class Conversation
     public string AgentId { get; set; } = string.Empty;
     public string UserId { get; set; } = string.Empty;
     public string Title { get; set; } = string.Empty;
+    public string Dialog { get; set; }
```

Comment:
Can we seperate Dialog to another collection? It will be optimized if UI only needs the conversation list.